### PR TITLE
Add docs for --headless

### PR DESCRIPTION
**Pull request by tarjei**
This was the best place I found for them .)

## Proposed changes
Document chrome headless mode

## Types of changes

[//]: # (What types of changes does your code introduce to WebdriverIO?)
[//]: # (_Put an `x` in the boxes that apply_)

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation

## Checklist

[//]: # (_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._)

- [ ] I have read the [CONTRIBUTING](https://github.com/webdriverio/webdriverio/blob/master/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

## Further comments

[//]: # (If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...)

### CODE REVIEW

**Comment by christian-bromann** [comment]
ping @tarjei

**Comment by tarjei** [comment]
@timbru31 fixed! Thanks for pointing that out.

File: docs/guide/testrunner/configurationfile.md [hunk 1]
```diff
@@ -78,7 +78,13 @@ exports.config = {
     // https://docs.saucelabs.com/reference/platforms-configurator
     //
     capabilities: [{
-        browserName: 'chrome'
+        browserName: 'chrome',
+        // to run chrome headless you can use the --headless flag, --disable-gpu is also required for now.
```
Comment (christian-bromann) [inline docs/guide/testrunner/configurationfile.md]: @tarjei one tiny thing: I think we can remove this line because you already mention the same below

Comment (tarjei) [inline docs/guide/testrunner/configurationfile.md]: Hmm, maybe say: 
`// to run chrome headless  --disable-gpu is also required for now.`

?

Comment (christian-bromann) [inline docs/guide/testrunner/configurationfile.md]: Then we still have `to run chrome headless` twice. Can we have just one comment about running chrome headless?

**Comment by tarjei** [comment]
@christian-bromann thanks for pointing out the duplication. I think it is better now.

File: docs/guide/testrunner/configurationfile.md [hunk 2]
```diff
@@ -79,10 +79,10 @@ exports.config = {
     //
     capabilities: [{
         browserName: 'chrome',
-        // to run chrome headless you can use the --headless flag, --disable-gpu is also required for now.
         chromeOptions: {
-        // to run in headless mode:
-        // See https://developers.google.com/web/updates/2017/04/headless-chrome
+        // to run chrome headless:
+        // --disable-gpu is also required for headless,
```
Comment (christian-bromann) [inline docs/guide/testrunner/configurationfile.md]: why `is also` here? Why not just
```js
// to run chrome headless the following Chrome flags are required
// (see https://developers.google.com/web/updates/2017/04/headless-chrome)
```

**Pull request merged**